tests/test_base.py

```diff
@@ -570,5 +570,25 @@ def test_finder(self):
                 finder.find('non_existing_key')
 
 
+class PathlibTestCase(unittest.TestCase):
```

Comment:
I would rather move these tests closer to other related tests:

1. "paths accepted as config files: "between `test_from_directory_and_files` and `test_from_globs` around here: https://github.com/Polyconseil/getconf/blob/f458ef81088dc3b66ffffcb5a2eaad2e2062c460/tests/test_base.py#L146

2. "paths accepted as default str values": in this existing function: https://github.com/Polyconseil/getconf/blob/f458ef81088dc3b66ffffcb5a2eaad2e2062c460/tests/test_base.py#L245

3. "paths accepted as default list values": in this other existing function: https://github.com/Polyconseil/getconf/blob/f458ef81088dc3b66ffffcb5a2eaad2e2062c460/tests/test_base.py#L280